examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,122 @@
+"""
+Demonstrate basic loading and plotting of cortical surface data
+===============================================================
+The dataset that is a subset of the enhanced NKI Rockland sample
+(http://fcon_1000.projects.nitrc.org/indi/enhanced/, Nooner et al, 2012)
+
+Resting state fMRI scans (TR=645ms) of 102 subjects were preprocessed
+(https://github.com/fliem/nki_nilearn) and projected onto the Freesurfer
+fsaverage5 template (Dale et al, 1999, Fischl et al, 1999). For this example
+we use the time series of a single subject's left hemisphere.
+
+The Destrieux parcellation (Destrieux et al, 2010) in fsaverage5 space as
+distributed with Freesurfer is used to select a seed region in the posterior
+cingulate cortex.
+
+Functional connectivity of the seed region to all other cortical nodes in the
+same hemisphere is calculated using Pearson product-moment correlation
+coefficient.
+
+The :func:`nilearn.plotting.plot_surf_stat_map` function is used
+to plot the resulting statistical map on the (inflated) pial surface.
+
+See :ref:`plotting` for more details.
```

Comment:
It would be useful to add a see also pointing to the other seed-based connectivity http://nilearn.github.io/auto_examples/03_connectivity/plot_seed_to_voxel_correlation.html and add a see also on the other example pointing here.